.github/workflows/launch-quickstart-mongodbatlas.yml

```diff
@@ -0,0 +1,153 @@
+name: Launch Quickstart 
+on:
+  workflow_dispatch:
+    inputs:
+      region: 
+        description: "AWS region"
+        default: "us-east-1"
+        required: true
+      projectName:
+        description: "Name"
+        default: "atlas-cfn-quickstart"
+        required: true
+      instanceSize:
+        description: "Instance size, defaults to M10"
+        default: "M10"
+        required: true
+      databaseusername:
+        description: "Database username"
+        default: "atlas"
+        required: true
+      databasepassword:
+        description: "Database password"
+        default: "atlas123"
```

Comment:
I'd suggest not putting any default to force it, otherwise people could use the default and potentially open themselves up.